Digital identity blog (#117)
ippontech/blog-usa: posts/hacking-the-gordian-knot-of-digital-identity.md

That's right folks, one day you may be able to move and _only change your address in one place_. All it takes is a little elbow grease and `trust in numbers`.

# Hacking the Gordian Knot

![Flying Spaghetti Monster](https://raw.githubusercontent.com/misterzero/blog-usa/master/images/2019/03/identity-spaghetti-monster.jpg)

There is no single technical panacea for self-sovereign identity. You can't just "put it on the blockchain". As Vitalek Buterin has pointed out, blockchains only care about accounts, and someone with sufficient funds can create thousands of "[sock puppets](https://www.coindesk.com/understanding-the-radicalxchange-movement-and-its-cypherpunk-appeal)" to overpower democratic votes. The solutions require a bit more nuance.

The good news is that there are several digital identity solutions, with varying degrees of decentralization and organizational transparency, which may fit our use cases. There is also a [decentralized digital identity standard](http://identity.foundation/) emerging which will allow for interoperability and portability.

> I hope you have a better grasp of some reasons you may want to sonsider decentralized federated identity solutions, and a basic understanding of the tenets of self-sovereign identity. In my next post I will dig in to the emerging DID standards and the platforms providing federated authentication.

Reviewer: ```suggestion
I hope you have a better grasp of some reasons you may want to consider decentralized federated identity solutions, and a basic understanding of the tenets of self-sovereign identity. In my next post I will dig into the emerging DID standards and the platforms providing federated authentication.
```

# Self-Sovereign Identity

So far we've discussed the importance of freeing our identities from Big Bad Tech. There is a rising tide of decentralist thinkers (_raises hand_) who believe there is also a case for separation if identity from nation-states, or [self-sovereign identity](https://bitsonblocks.net/2017/05/17/gentle-introduction-self-sovereign-identity/).

![Self-Sovereignty](https://raw.githubusercontent.com/misterzero/blog-usa/master/images/2019/03/identity-self-sovereignty.jpg)

I think the simplest articulation of self-sovereignty is that your home country may rightfully control your rights as a _citizen_, but every human has the right to control their own proof of _identity_. Let's unpack that a bit and look at some examples to illustrate the differences.

Just like an online retailer or social media platform, your local, state, and national government probably has some stake in data about you that results from civic interaction (or "imperial entanglements"). Examples are the date of your birth or first entry into the country, any visas or federal licenses that you may have been granted. These agencies are the source of truth for these facts about you.

> Otther agencies or nations may have similar claims to facts about you. Maybe you are a dual passport holder, or one of the growing class of [digital nomads](https://www.reddit.com/r/digitalnomad/). Some countries may have requirements around your status in your home country, but generally they just want to know _who you are_.

Reviewer 2: ```suggestion
Other agencies or nations may have similar claims to facts about you. Maybe you are a dual passport holder, or one of the growing class of [digital nomads](https://www.reddit.com/r/digitalnomad/). Some countries may have requirements around your status in your home country, but generally they just want to know _who you are_.
```

Reviewer 2: 🤦‍♂️

 * Healthcare aid workers in refugee camps can access a family's vaccination records using a parent's fingerprint
 * Healthcare providers in general can access a patient's medical records, as shared by the client, and verification of payment for services, without having to know their name or address
 * Everyone can access a global directory of healthcare providers, search by geography, specialty, and client ratings, and make appointments
 * Digital ID holders are eligible to participate in a global emergency notification service, like 911 in the US, using a smartphone from any mobile or wifi network
 * Citizens can cast votes on proposed legislation or platform issues using digital ID
 * Digital ID holders can share their personal information e.g. contact information, billing information, KYC data for financial transactions, locked under a single secure identity

That's right folks, one day you may be able to move and _only change your address in one place_. All it takes is a little elbow grease and `trust in numbers`.

# Hacking the Gordian Knot

> ![Flying Spaghetti Monster](https://raw.githubusercontent.com/misterzero/blog-usa/master/images/2019/03/identity-spaghetti-monster.jpg)

Reviewer 2: This image has the background. If you wanted the transparent version, checkout 

https://i.pinimg.com/originals/69/09/ef/6909ef0d0bd64ed91aa17a08363edde1.png
![](https://i.pinimg.com/originals/69/09/ef/6909ef0d0bd64ed91aa17a08363edde1.png)

or https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/intermediary/f/2ab79e5b-2df6-4ad2-8567-6ab21cd0efcf/dblpo75-593bb595-ba60-4ef5-a4ba-b7143e2b0597.png/v1/fill/w_395,h_350,strp/updated_flying_spaghetti_monster____by_mizushimatakara_dblpo75-350t.png
![](https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/intermediary/f/2ab79e5b-2df6-4ad2-8567-6ab21cd0efcf/dblpo75-593bb595-ba60-4ef5-a4ba-b7143e2b0597.png/v1/fill/w_395,h_350,strp/updated_flying_spaghetti_monster____by_mizushimatakara_dblpo75-350t.png)

Reviewer 2: not critical

There is no single technical panacea for self-sovereign identity. You can't just "put it on the blockchain". As Vitalek Buterin has pointed out, blockchains only care about accounts, and someone with sufficient funds can create thousands of "[sock puppets](https://www.coindesk.com/understanding-the-radicalxchange-movement-and-its-cypherpunk-appeal)" to overpower democratic votes. The solutions require a bit more nuance.

The good news is that there are several digital identity solutions, with varying degrees of decentralization and organizational transparency, which may fit our use cases. There is also a [decentralized digital identity standard](http://identity.foundation/) emerging which will allow for interoperability and portability.

> I hope you have a better grasp of some reasons you may want to sonsider decentralized federated identity solutions, and a basic understanding of the tenets of self-sovereign identity. In my next post I will dig in to the emerging DID standards and the platforms providing federated authentication.

Reviewer 2: ```suggestion
I hope you have a better grasp of some reasons you may want to consider decentralized federated identity solutions, and a basic understanding of the tenets of self-sovereign identity. In my next post I will dig in to the emerging DID standards and the platforms providing federated authentication.
```

Reviewer 2: 🤦‍♂️

Reviewer: 👀
